resources/authgear/templates/en/translation.json

```diff
@@ -1071,5 +1071,6 @@
   "v2-account-linking-subtitle--login": "Seems like you have signed up before. Log in to an existing account below.",
   "v2-account-linking-by-email": "Login to {IdentityDisplayName}",
   "v2-account-linking-by-phone": "Login to {IdentityDisplayName}",
-  "v2-account-linking-by-username": "Login to {IdentityDisplayName}"
+  "v2-account-linking-by-username": "Login to {IdentityDisplayName}",
+  "v2-settings-edit-profile": "<a href=\"{href}\">Edit Profile</a>"
```

Comment:
Our SOP is en, zh-hant and zh-hans are inputted by us. Please write those translation